### PR TITLE
Easier imports

### PR DESCRIPTION
**Pull request by jacobsimon**
I added Button to `index.js` and changed this file to use the ES6 export syntax. This will simplify the import syntax for consumers -- for example, you can write:
```javascript
import {Button} from "clever-components"; // ES6
```
or
```javascript
var Button = require("clever-components").Button // ES5
```
instead of writing
```javascript
var Button = require("clever-components/src/Button/Button").default
```
Todo: Confirm that this works outside of Chrome.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,2 @@
-module.exports = {
-  Modal: require("./src/Modal/Modal")
-}
+export { default as Modal } from './src/Modal/Modal';
+export { default as Button } from './src/Button/Button';
```
Comment (rgarcia) [inline index.js]: could this be `export Button from './src/Button/Button'`?

**Comment by rgarcia** [comment]
lgtm!

File: index.js (hunk 1 above)
Comment (jacobsimon) [inline index.js]: https://github.com/Clever/components/pull/4/commits/5e1a73994394dcd8de9671c519aa906dd1c6c571

**Comment by rgarcia** [comment]
@jacobsimon looks like this breaks the tests `Error: Cannot find module '!style!css!less!./Modal.less'`

**Pull request merged**